helmfile.d/upstream/falcosecurity/falco/BREAKING-CHANGES.md

```diff
@@ -9,6 +11,21 @@
     - [Falco Images](#drop-support-for-falcosecurityfalco-image)
     - [Driver Loader Init Container](#driver-loader-simplified-logic)
 
+## 6.0.0
+
+### Falco Talon configuration changes
+
+The following backward-incompatible changes have been made to `values.yaml`:
+- `falcotalon` configuration has been renamed to `falco-talon`
+- `falcotalon.enabled` has been renamed to `responseActions.enabled`
+
+## 5.0.0
+### Default Falco Image
+**Starting with version 5.0.0, the Helm chart now uses the default Falco container image, which is a distroless image without any additional tools installed.**
+Previously, the chart used the `debian` image with the several tools included to avoid breaking changes during upgrades. The new image is more secure and lightweight, but it does not include these tools.
+
+If you rely on some tool—for example, when using the `program_output` feature—you can manually override the `image.tag` value to use a different image flavor. For instance, setting `image.tag` to `0.41.0-debian` will restore access to the tools available in the Debian-based image.
```

Comment:
Should we also consider changing to the distroless image? I don't think we need any of the extra tools that might be available. But I also do not really know what tools they have added to the debian image, so I could be missing something important.

Reply:
Yeah and we anyway had to do several other changes in order to get a core dump, so then we could have switched to the debian image if necessary as well.
As you say, ephemeral containers can also be a way to get more tooling. Although I don't know how well tested that is for us yet, it might run into some issues with some gatekeeper policies.
I'm still interested in trying the distroless image. But it does not have to be in this PR.